src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue

```diff
@@ -534,6 +534,7 @@ $row-arrow-width: 2.5rem;
 		width: rem-calc(100);
 		height: rem-calc(100);
 		pointer-events: none;
+		z-index: 1;
```

Comment:
This was showing up in the VM...maybe not an issue once compiled fully. Just a safeguard